### PR TITLE
OKTA-566356 : g3 : Fixing spec test for SIW v3 polling

### PR DESCRIPTION
**Pull request by glenfannin-okta**
## Description:

The purpose of this PR is to re-enable the spec parity test for polling.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-566356](https://oktainc.atlassian.net/browse/OKTA-566356)

### CODE REVIEW

File: test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js [hunk 1]
```diff
@@ -536,18 +536,20 @@ test.meta('v3', false)
       record => record.response.statusCode === 200 &&
         record.request.url.match(/poll/)
     )).eql(3);
+    logger.clear();
```
Comment (shawyu-okta) [inline test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js:539]: Why did we need to add these now when before it worked without them?

**Review by shawyu-okta — APPROVED** [review]

File: test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js (hunk 1 above)
Comment (glenfannin-okta) [reply test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js:539]: Actually, I am not totally sure lol, what was happening was the logger object contained a bunch of the previous logs which caused the final assertion to have 16+ messages in them. After clearing the logs (after each log assertion), it brought the final assertion down to what was expected which was 5 log messages (the additional one is caused by auth-js). But I am not sure what v2 is doing that made it just completely ignore the others 🤔 its super perplexing 😢

Comment (leemchale-okta) [reply test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js:539]: so this ticket https://oktainc.atlassian.net/browse/OKTA-587189 will fix that 1 extra poll, but we still need to clear the logger each time?

Comment (glenfannin-okta) [reply test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js:539]: Yep, clearing the logger prevents it from containing log messages from earlier parts of the test.

Comment (glenfannin-okta) [reply test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js:539]: Without it, the last assertion has double the amount of logs in it and has a large variance compared to v2.

**Pull request merged**